channeldb/migration14/legacy_decoding.go

```diff
@@ -42,12 +44,22 @@ func deserializeLogUpdatesLegacy(r io.Reader) ([]LogUpdate, error) {
 	logUpdates := make([]LogUpdate, numUpdates)
 	for i := 0; i < int(numUpdates); i++ {
 		err := ReadElements(r,
-			&logUpdates[i].LogIndex, &logUpdates[i].UpdateMsg,
+			&logUpdates[i].LogIndex,
 		)
 		if err != nil {
+			fmt.Println("No more for me")
 			return nil, err
 		}
+
+		updateMsg, err := lnwire.ReadMessage(r, 0)
+		if err != nil {
+			fmt.Println("tried to read everything")
+			return nil, err
+		}
```

Comment:
In any case, I wouldn't squash this with the copy/paste commit. At least make a PURE MOVE commit (not necessarily building) and do all required changes in a separate commit. Otherwise it becomes hard to review.

Reply:
Moved to a new commit.